src/ate/test_programs/dut_lib/dut_lib.cc

```diff
@@ -23,7 +24,14 @@ std::unique_ptr<DutLib> DutLib::Create(void) {
 absl::Status DutLib::DutInit(const std::string& fpga,
                              const std::string& fpga_bitstream) {
   LOG(INFO) << "in DutLib::DutInit";
-  OtLibFpgaInit(fpga.c_str(), fpga_bitstream.c_str());
+  transport_ = OtLibFpgaInit(fpga.c_str(), fpga_bitstream.c_str());
```

Comment:
You might want to make creating the transport and FPGA Init separate calls to rust.  That would allow you to get the transport pointer in `DutLib::Create` and then have `Init`, `LoadSramElf` and other operations be able to use the transport.  By not forcing an FPGA Init each time, you'll enable other flows that might be useful in a debug situation.

Reply:
Ah yes, I am currently working on this cleanup in a follow up PR; I will post it soon.

Reply:
Fixed in #96.